jira/src/main/kotlin/gropius/sync/jira/JiraSync.kt

```diff
@@ -182,14 +184,37 @@ final class JiraSync(
         return issueDataService.findByImsProject(imsProject.rawId!!)
     }
 
+    /**
+     * Map list of User to GropiusUser
+     * @param users The list of users mixed of IMSUser and GropiusUser
+     * @return The list of GropiusUser
+     */
+    private suspend fun gropiusUserList(users: List<User>): List<GropiusUser> {
```

Comment:
no place anywhere in the common sync module where this could be?
looks duplicated it github/jira

Reply:
59808ea880a17da3dfd2ab6a3223f2a8bf8532ba